diagnostic.md

```diff
@@ -5,74 +5,86 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+To store data remotely and to abstract data from the user (only retrieve or
+show specific relevent data to user) and to allow multiple network connections.
+Something to do with security as well.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Rachel said something about us using views still but in a different way than the
+"MVC" way. I dont know what it is though.
 ```
```

Comment:
If you revisit the SPA study, it's basically that. Not using the SPA standard is almost always a terrible decision. We don't use the traditional MVC view, we manage the view layer differently.